memory/history/simple.go

```diff
@@ -0,0 +1,55 @@
+package history
+
+import "github.com/tmc/langchaingo/schema"
+
+// Statically assert that the types implement the interface.
+var _ schema.ChatMessageHistory = (*SimpleChatMessageHistory)(nil)
+
+// SimpleChatMessageHistory is a struct that stores chat messages.
+type SimpleChatMessageHistory struct {
```

Comment:
the `SimpleChatMessageHistory` is the equivalent to the original In-memory implementation. Should we just call it InMemoryChatMessageHistory instead ?

Reply:
I kept the "simple" prefix because the in-memory `Memory` implementation is also referenced as "simple" here:
https://github.com/tmc/langchaingo/blob/main/memory/simple.go#L9